nodebuilder/tests/fuzz_test.go

```diff
@@ -0,0 +1,201 @@
+//go:build da || integration
+
+package tests
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/libp2p/go-libp2p/core/host"
+	"github.com/libp2p/go-libp2p/core/peer"
+	"github.com/stretchr/testify/require"
+
+	"github.com/celestiaorg/celestia-node/nodebuilder/da"
+	"github.com/celestiaorg/celestia-node/nodebuilder/node"
+	"github.com/celestiaorg/celestia-node/nodebuilder/tests/swamp"
+	"github.com/celestiaorg/celestia-node/share"
+)
+
+var namespace, _ = share.NewBlobNamespaceV0([]byte("namespace"))
+
+type daModuleCorpus struct {
+	Blobs [][]byte `json:"blobs"`
+}
+
+func FuzzDaModule(f *testing.F) {
+	// 1. Retrieve the fuzz corpra.
+	corpraPath := filepath.Join("testdata", "bytes-corpra", "*.bytes")
```

Comment:
`bytes-submit` ?

Reply:
Sorry I changed the path name while the fuzzer was running during its 8 hours. I'll update that, thanks!